src/components/ArticleCard.jsx

```diff
@@ -16,7 +22,9 @@ function ArticleCard({article}) {
             <Card.Text>
             Topic: {article.topic}
             </Card.Text>
-            <Button variant="primary">Read More...</Button>
+            <Link to={`/articles/${article.article_id}`}> 
```

Comment:
Nice use of Link to direct the user to the appropriate route for the article they have selected

Reply:
Thanks